esphome/const.py

```diff
@@ -13,6 +13,7 @@
 PLATFORM_HOST = "host"
 PLATFORM_LIBRETINY_OLDSTYLE = "libretiny"
 PLATFORM_RP2040 = "rp2040"
+PLATFORM_LN882X = "ln882x"
```

Comment:
Alphabetical order =)
```suggestion
PLATFORM_LN882X = "ln882x"
PLATFORM_RP2040 = "rp2040"
```